src/lib/alpha.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-misleading-character-class */
```

Comment:
I prefer disabling eslint for only the lines where it is needed. Do we have many regexes where we use 'misleading' characters in this file?

Reply:
yeah we do in the file. This is just the lazy stuff